src/Components/Modal/edit-profile-modal.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import { BsX } from 'react-icons/bs';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchEditUser } from '../../features/users/userSlice';
+
+export function EditProfile({ setShowModal, setImg, img }) {
+
+  const dispatch = useDispatch();
+  const userProfileData = useSelector(state => state.userProfile);
+  const authUserData = useSelector(state => state.user);
+  const [userData, setUserData] = useState({...authUserData.user});
+  const token = authUserData.token;
```

Comment:
since you are only using the `token` here.. you can destructure it if you want. 
e.g. 
```
const {token} = useSelector(state => state.user)
```